README.rst

```diff
@@ -20,18 +19,20 @@ install
 -------
 
 ::
+
     $ pip install pytest-external-blockers
 
 
 use
 ---
 
-::
+.. code-block:: python
+
     import os
     import pytest
     from .issues import get_tracker
 
-    pytestmark = pytest.mark.skipifif(
+    pytestmark = pytest.mark.skipif(
```

Comment:
Is this supposed to be here? It seems that the `bugtracker` fixture already "blocks" the test if the bucktracker is not available.